examples/grpc/client.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
```

Comment:
Probably, we can move this example in a `grpc/static_codegen` folder and add another example, under `grpc/dynamic_codegen` which can be the dynamic code generation (generated at runtime using `Protobuf.js`) variant of gRPC example. WDYT?